packages/website/src/components/ItemSidebar.tsx

```diff
@@ -0,0 +1,31 @@
+import { VscPackage } from 'react-icons/vsc';
+import { generateIcon } from '~/util/icon';
+import type { getMembers } from '~/util/parse.server';
+
+export interface ItemListProps {
+	packageName: string;
+	data: {
+		members: ReturnType<typeof getMembers>;
+	};
+}
+
+export function ItemSidebar({ packageName, data }: ItemListProps) {
+	return (
+		<div className="flex flex-col px-7">
+			<div className="flex items-center content-center">
+				<h1 className="px-2 font-mono">
```

Comment:
```suggestion
			<div>
				<h1 className="px-2 font-mono flex items-center content-center">
```

Should probably apply this here, otherwise the icons and the text are not vertically centered to each other.

Reply:
Reason: The div only has one child, while the `h1` is the one that has 2 that need to be centered to each other.